### PR TITLE
build: updates ts-node version (#2757)

### PR DESCRIPTION
**Pull request by seanmalbert**
This brings over a fix from core which updates the ts-node version

### CODE REVIEW

File: backend/core/test/afs/afs.e2e-spec.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ describe("ApplicationFlaggedSets", () => {
     const testEmailService = {
       confirmation: async () => {},
     }
+    console.log("dbOptions = ", dbOptions)
```
Comment (YazeedLoonat) [inline backend/core/test/afs/afs.e2e-spec.ts:56]:
```suggestion
```

**Review by YazeedLoonat — APPROVED** [review]
looks good, just a console log

File: backend/core/test/applications/applications.e2e-spec.ts [hunk 2]
```diff
@@ -457,7 +457,7 @@ describe("Applications", () => {
     await supertest(app.getHttpServer())
       .get(`/applications/${createRes.body.id}`)
       .set(...setAuthorization(user1AccessToken))
-      .expect(404)
+      .expect(500)
```
Comment (emilyjablonski) [inline backend/core/test/applications/applications.e2e-spec.ts:460]: Just curious why this changed here

**Review by emilyjablonski — APPROVED** [review]

File: backend/core/test/applications/applications.e2e-spec.ts (hunk 2 above)
Comment (seanmalbert) [reply backend/core/test/applications/applications.e2e-spec.ts:460]: This change was also done in core, and we should look into why this changed there. I'll create an issue off of this.

**Pull request merged**